src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildWithATargetPlatform.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Xunit;
+using FluentAssertions;
+using Xunit.Abstractions;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+using System.IO;
+using System;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantToBuildWithATargetPlatform : SdkTest
+    {
+        public GivenThatWeWantToBuildWithATargetPlatform(ITestOutputHelper log) : base(log)
+        {
+        }
+
+        [Theory]
+        [InlineData("netcoreapp3.1", ".NETCoreApp", "v3.1", "Windows", "7.0")] // Default values pre-5.0
+        [InlineData("net5.0", ".NETCoreApp", "v5.0", "Windows", "7.0")]
```

Comment:
The expected platform should not be set to Windows in this case, right?  Or if it currently is set to Windows, then I think we should take the fix from Master to only set it to Windows for Windows Forms / WPF apps, and apply that to preview 8 also.

Reply:
We reverted the change that disabled the legacy defaults for preview 8 when we merged preview 7 into preview 8 earlier this week, here: #12577

Reply:
As discussed offline, this change has been added to the other preview 8 PR I have out, which has carries the test attributes this change relies on: https://github.com/dotnet/sdk/pull/12613/commits/a8bbd938a566d77943ad758b4ec5e4dd80fda7b0